cartography/data/indexes.cypher

```diff
@@ -149,6 +149,11 @@ CREATE INDEX ON :GCPMonitoringMetricDescriptor(id);
 CREATE INDEX ON :GCPMonitoringNotificationChannel(id);
 CREATE INDEX ON :GCPMonitoringUptimeCheckConfig(id);
 CREATE INDEX ON :GCPDataprocCluster(id);
+CREATE INDEX ON :GCPBackendBucket(id);
+CREATE INDEX ON :GCPGlobalBackendService(id);
```

Comment:
We should have just 1 Backend Service object with an attribute whether it's global or regional. Type of backend service.